dist/ansible/scripts/ovn-setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+  
+# Create a configmap of the ovn configuration information
+# This script runs on the ovn-master node on the cluster
+# This is run as part of installation after the cluster is
+# up and before the ovn daemonsets are created.
+
+health=$(oc get --raw /healthz/ready)
+if [ $health != 'ok' ]
+then
+  echo " cluster not ready"
+  exit 1
+fi
+
+svcacct=$(oc get serviceaccount | grep ovn)
+if [ $? -ne 0 ]
+then
+  oc create serviceaccount ovn
+  oc adm policy add-cluster-role-to-user cluster-admin -z ovn
+  oc adm policy add-scc-to-user anyuid -z ovn
+fi
+
+# get api server
+api=$(oc get po -n kube-system -o wide | gawk '/master-api/{ print $7 }')
+apiserver=https://${api}:8443
+
+# get the API token.
+token=$(oc sa get-token ovn)
+
+# get the network cidr (master-config.yaml clusterNetworkCIDR)
+#net_cidr=$(oc get clusternetwork | gawk '/default/{ print $2 }' | gawk -F ":" '{ print $1}')
+net_cidr=10.128.0.0/14
+
+# get the service subnet cidr (master-config.yaml servicesSubnet)
+#cidr=$(oc get clusternetwork | gawk '/default/{ print $2 }' | gawk -F ":" '{ print $1}')
```

Comment:
Same here; just remove the oc comment line.

Reply:
OK